articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.md

```diff
@@ -77,38 +77,38 @@ Create an `Owner` entity with the following attributes:
 
 ## Specify the relationships between entities
 
-Specifying [relationships](https://en.wikipedia.org/wiki/Class_diagram#Instance-level_relationships) between entities in the Intent Architect modelers are initiated by right-clicking an entity in the visual diagram, then clicking the `New Association` option and then completed by clicking on the other class which you want to be the target of the relationship.
+Specifying [relationships](https://en.wikipedia.org/wiki/Class_diagram#Instance-level_relationships) between entities in the Intent Architect modelers can be initiated by right-clicking an entity in the visual diagram, then clicking the `New Association` option and then completed by clicking on the other class which you want to be the target of the relationship.
 
 Create an association between `Owner` and `Pet`:
 
-- Right-click on `Owner` entity.
+- Right-click on the `Owner` entity.
 - Click on the `New Association` option.
 - Click on the `Pet` entity which completes the creation of the association.
 - In the property pane (in the bottom-right corner of the screen) within the `Target End` section ensure that `Is Collection` is checked.
 - In the property pane within the `Source End` section ensure that `Is Collection` is *un*checked and that `Navigable` is checked.
 
 <p><video style="max-width: 100%" muted="true" loop="true" autoplay="true" src="videos/add-owner-pet-association.mp4"></video></p>
 
-Setup the `Visit` entity to be as follows:
+Setup a `Visit` entity to be as follows:
 
-- Add an association between `Pet` and `Visit`:
+- Add an association from `Pet` to `Visit`:
```

Comment:
Because the source of the association matters, I think it's better to use "from" and "to" here.